app/error/template.njk

```diff
@@ -0,0 +1,27 @@
+{% extends '../includes/layout.njk' %}
+{% from 'components/error-summary/macro.njk' import errorSummary %}
+{% from 'components/back-link/macro.njk' import backLink %}
+
+{% block body %}
+<div class="nhsuk-width-container nhsuk-u-margin-top-4 nhsuk-u-margin-bottom-4 bc-c-error-container nhsuk-u-padding-4">
+  <h1 class="nhsuk-heading-l nhsuk-u-padding-left-3" data-test-id="error-page-title">Error: {{ message }}</h1>
+    <div class="nhsuk-u-padding-top-5 nhsuk-grid-column-one-third" data-test-id="go-to-home-page-link"> 
+      {{ backLink({
+        "href": "/",
+        "text": "Go to Home Page"
+      }) }}
+    </div>
+    <div class="nhsuk-u-padding-top-5 nhsuk-grid-column-one-third" data-test-id="go-to-all-solutions-link"> 
+      {{ backLink({
+        "href": "/solutions/all",
+        "text": "Go to All Solutions"
+      }) }}
+    </div>
+    <div class="nhsuk-u-padding-top-5 nhsuk-grid-column-one-third" data-test-id="go-to-foundation-solutions-link"> 
+      {{ backLink({
+        "href": "/solutions/foundation",
+        "text": "Go to Foundation Solutions"
+      }) }}
+    </div>
```

Comment:
I don't understand why there are 3 backlinks?

Reply:
because it is a generic error page. can be changed just did what I thought as there was no guidance

Reply:
pull the branch and take a look

Reply:
Just move these back links above the Error message